packages/react/src/primitives/SwitchField/__tests__/SwitchField.test.tsx

```diff
@@ -106,38 +102,21 @@ describe('Switch Field', () => {
   });
 
   describe('Input', () => {
-    let updateControlledValue;
-    const ControlledSwitch = () => {
-      const [isChecked, setIsChecked] = React.useState(true);
-      const changeFunction = (e) => {
-        setIsChecked(e.target.checked);
-      };
-      updateControlledValue = setIsChecked;
-
-      return (
-        <SwitchField
-          label={label}
-          isChecked={isChecked}
-          onChange={changeFunction}
-        />
-      );
-    };
```

Comment:
This test component was extraneous, the desired behavior can be achieved using the `rerender` function returned from calling `render`